scss/_base_typography-definitions.scss

```diff
@@ -204,9 +204,10 @@
   }
 
   %table-header-label {
-    @extend %x-small-text;
-
-    text-transform: uppercase;
+    @extend %common-default-text-properties;
+    font-variant-caps: all-small-caps;
```

Comment:
@lyubomir-popov should these styles be for table header only?  or should they be used in other places where we capitalise text (like the p-text-x-small--capitalised)?